coding-concepts.js

```diff
@@ -37,23 +37,23 @@ const languages = ["JavaScript", "Ruby", "Python", "C++"]
 const index = 1
 // console.log(languages[index])
 
-// a) Your answer:
-// b) Verify and explain:
+// a) Your answer: The string "Ruby"
+// b) Verify and explain: The string "Ruby" is at the index of 1 in the array and thhe console is logging the variable languages with a index of the variable index which is set to 1
 
 
 // --------------------4) What will this log?
 
 const weekendDays = ["saturday", "sunday"]
 // console.log(weekendDays.toUpperCase())
 
-// a) Your answer:
-// b) Verify and explain:
+// a) Your answer: ["SATURDAY", "SUNDAY"]
+// b) Verify and explain: I was wrong since .toUpperCase() is a method for strings not arrays. You got me there!
 
 
 // --------------------5) What will this log?
 
 const dataTypes = ["number", "string", "Boolean"]
 // console.log(typeof dataTypes.length)
 
-// a) Your answer:
-// b) Verify and explain:
+// a) Your answer: Either a variable || a number
+// b) Verify and explain: Both wrong and right? I was thinking it might find the typeof dataTypes which is a variable or since the .length is finding the length of the array it would equal 3, which is a number.
```

Comment:
interestingly enough "variable" is not a data type. Variables only hold data with their own types. So their would be a conflict if variables had their own type. That one took me a while to grasp.